### ISSUE TITLE
connectToScorebot not working

### ISSUE THREAD

**Issue by Felipevieira2** [gigobyte/HLTV#386]
I am trying to use connectToScoreBOt and no data is returned in my console.log (), in my code below, am I making a mistake?

```
	HLTV.connectToScorebot({id: 2346282, onScoreboardUpdate: (data, done) => {
		// if you call done() the socket connection will close.
		console.log(data)

		}, onLogUpdate: (data, done) => {
			console.log(data)
		}
	})	
```

**Comment by gigobyte** [comment]
Maybe HLTV updated their websocket communication, I'll label this as a bug until further investigation.

**Comment by gigobyte** [comment]
Fixed in 2.23.0
